endtoend/README.md

```diff
@@ -10,8 +11,8 @@ In order to "evaluate" the state of the beacon chain while the E2E is running, t
 Evaluators have 3 parts, the name for it's test name, a `policy` which declares which epoch(s) the evaluator should run, and then the `evaluation` which uses the beacon chain API to determine if the beacon chain passes certain conditions like finality.
 
 ## Current end-to-end tests
-* Minimal Config - 4 beacon nodes, 64 validators, running for 5 epochs
-* Demo Config - 4 beacon nodes, 64 validators, running for 5 epochs
+* Minimal Config - 4 beacon nodes, 64 validators, running for 6 epochs
+* ~~Demo Config - 2 beacon nodes, 16,384 validators, running for 5 epochs~~ Disabled for now
```

Comment:
Any reasons for the `~~`??

```suggestion
* Demo Config - 2 beacon nodes, 16,384 validators, running for 5 epochs~~ Disabled for now
```

Reply:
Nvm it's the tilting the font

Reply:
Just figured to scratch it out since it's disabled at the moment.